src/gitgraph.js

```diff
@@ -307,6 +315,40 @@
   };
 
   /**
+   * A callback for each commit
+   *
+   * @callback commitCallback
+   * @param {Commit} commit - A commit
+   * @param {boolean} mouseOver - True, if the mouse is currently hovering over the commit
+   */
+
+  /**
+   * Hover event on commit dot
+   *
+   * @param {MouseEvent} event - Mouse event
+   * @param {commitCallback} callbackFn - A callback function that will be called for each commit
+   *
+   * @self Gitgraph
+   **/
+  GitGraph.prototype.applyCommits = function(event, callbackFn) {
+    // Fix firefox MouseEvent
+    event.offsetX = event.offsetX ? event.offsetX : event.layerX;
```

Comment:
@terhechte > refactoring looks fine, but what happens to the Firefox `if` guard? :smiley: 

```javascript
if ( typeof InstallTrigger !== "undefined" )/* == (is Firefox) */ {       		
  event.offsetX = event.offsetX ? event.offsetX : event.layerX;		
  event.offsetY = event.offsetY ? event.offsetY : event.layerY;		
  event.x = event.x ? event.x : event.clientX;		
  event.y = event.y ? event.y : event.clientY;		
}
```